UnityClient/Assets/Scripts/TowerDefence/PlayerStats.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerStats : MonoBehaviour
+{
+    //public static int Money; 
+    
+    // public static int Lives;
+
+    public static int Rounds;
```

Comment:
`static` is strictly prohibited for use cases like this. It is brittle and not good design. Please redesign this to not be `static`. Thanks. Assuming this is public-facing data you might also want to consider this to be a property, or whatever fits your new design.

Reply:
If I remove this it doesn't work - if it works then its good for the prototype. Will think of a work around later